### ISSUE TITLE
Flow errors from the node_module package

### ISSUE THREAD

**Issue by ggregoire** [styled-components/styled-components#146]
Hi!

To start, thank you for `styled-components`, I really love your library. 👌

## Version
1.0.9

## Actual Behavior

Flow shows me several errors from the node_module package:

![screen shot 2016-10-28 at 13 16 01](https://cloud.githubusercontent.com/assets/591804/19817511/6d4a8536-9d11-11e6-8d89-320249cbe4fa.png)

I tried to ignore the package in my `.flowconfig`:

```
[ignore]
<PROJECT_ROOT>/node_modules/fbjs/.*
<PROJECT_ROOT>/node_modules/styled-components/.*
```

(the first line comes from https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-flow)

But then I got a new error from my own imports:

![screen shot 2016-10-28 at 13 17 43](https://cloud.githubusercontent.com/assets/591804/19817589/c5d9417e-9d11-11e6-8c06-41128ef10b0e.png)

Any idea how to remove these errors?

Thanks!

**Comment by relekang** [comment]
I think this will be fixed by #148

Meanwhile you might add a suppressing comment, [see the options docs for flow](https://flowtype.org/docs/advanced-configuration.html#options). Flow will error when the comment is no longer needed.

**Comment by mxstbr** [comment]
Closing in favor of #51 (which will be fixed by #148)
